dotfiles/.aliases

```diff
@@ -47,6 +47,10 @@ alias es="ember server"
 alias et="ember test"
 alias ets="ember test --server --no-launch"
 
+alias nom="rm -rf node_modules;npm cache clean; npm i"
+alias yom="rm -rf node_modules;yarn"
+alias bom="rm -rf bower_components;bower i"
```

Comment:
can we get a combined `nombom` and `yombom` that runs both commands? Probably just:

```
alias nombom="nom && bom"
alias yombom="yom && bom"
```